openfpga.sh

```diff
@@ -16,6 +16,12 @@ export OPENFPGA_SCRIPT_PATH="${OPENFPGA_PATH}/openfpga_flow/scripts"
 export OPENFPGA_TASK_PATH="${OPENFPGA_PATH}/openfpga_flow/tasks"
 if [ -z $PYTHON_EXEC ]; then export PYTHON_EXEC="python3"; fi
 
+
```

Comment:
This addition to PATH is no longer required. You may revert changes to this file.